index.js

```diff
@@ -419,7 +419,8 @@ module.exports.removeTagsFromBuffer = function(data) {
 
     let hSize = Buffer.from([data[framePosition + 6], data[framePosition + 7], data[framePosition + 8], data[framePosition + 9]])
 
-    if ((hSize[0] | hSize[1] | hSize[2] | hSize[3]) & 0x80) {
+    const isMsbSet = !!((hSize[0] | hSize[1] | hSize[2] | hSize[3]) & 0x80)
+    if (!isMsbSet) {

```

Comment:
I think the tests fail because it should be `isMsbSet` without the `!`

Reply:
Ah yes, you are right, I have been confused by the comment below as the comment was having a **not** in the wording! This should be fixed now.